components/opc-ua/src/Microsoft.Azure.IIoT.OpcUa.Protocol/src/Services/ClientServices.cs

```diff
@@ -423,222 +408,6 @@ public class ClientServices : IClientHost, IEndpointServices, IEndpointDiscovery
-                ApplicationUri = "urn:" + Utils.GetHostName() + ":Azure:IIoTOpcTwin",
-                CertificateValidator = new CertificateValidator(),
-                SecurityConfiguration = new SecurityConfiguration {
-                    ApplicationCertificate = new CertificateIdentifier {
-                        StoreType = effectiveAppCertStoreType,
-                        StorePath = effectiveOwnCertPath,
-                        SubjectName = "Azure IIoT OPC Twin"
-                    },
-                    TrustedPeerCertificates = new CertificateTrustList {
-                        StoreType = CertificateStoreType.Directory,
-                        StorePath = _configuration.TrustedCertPath
-                    },
-                    TrustedIssuerCertificates = new CertificateTrustList {
-                        StoreType = CertificateStoreType.Directory,
-                        StorePath = _configuration.IssuerCertPath
-                    },
-                    RejectedCertificateStore = new CertificateTrustList {
-                        StoreType = CertificateStoreType.Directory,
-                        StorePath = _configuration.RejectedCertPath
-                    },
-                    NonceLength = 32,
-                    AutoAcceptUntrustedCertificates = _configuration.AutoAcceptUntrustedCertificates,
-                    RejectSHA1SignedCertificates = false,
-                    AddAppCertToTrustedStore = false,
-                    MinimumCertificateKeySize = 1024
-                },
-                TransportConfigurations = new TransportConfigurationCollection(),
-                TransportQuotas = TransportQuotaConfigEx.DefaultTransportQuotas(),
-                ClientConfiguration = new ClientConfiguration {
-                    DefaultSessionTimeout = (int)sessionTimeout.TotalMilliseconds
-                }
-            };
-            applicationConfiguration.TransportQuotas.OperationTimeout = (int)operationTimeout.TotalMilliseconds;
-            return applicationConfiguration;
-        }
-
-        /// <summary>
-        /// Initialize the OPC UA Application's security configuration
-        /// </summary>
-        /// <returns></returns>
-        private async Task InitApplicationSecurityAsync() {
-
-            // update certificates validator
-            _appConfig.CertificateValidator.CertificateValidation +=
-                new CertificateValidationEventHandler(VerifyCertificate);
-            await _appConfig.CertificateValidator
-                .Update(_appConfig).ConfigureAwait(false);
-
-            // lookup for an existing certificate in the configured store
-            var ownCertificate = await _appConfig.SecurityConfiguration
-                .ApplicationCertificate.Find(true).ConfigureAwait(false);
-            if (ownCertificate == null) {
-                //
-                // Work around windows issue and lookup application certificate also on
```

Comment:
Is this logic still there somewhere?   And if not, why?   I remember this was important to fix issues where old certificates were in the store and at a certain point it would all fail until you "manually" fixed the store...   We'd want to avoid this in module unattended scenarios.

Reply:
this was a windows store specific workaround, I think is not necessary